ui/components/Text.tsx

```diff
@@ -42,6 +42,8 @@ const Text = styled.span<TextProps>`
 
   ${(props) => props.noWrap && "white-space: nowrap;"};
   ${(props) => props.titleHeight && "line-height: 1.75"};
+  //matches MuiIcon
+  transition: color 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
```

Comment:
Do we need transitions on ALL instances of the `Text` component? Can we use styled `Text` with transitions only where it's needed instead (nav links or whatever).

Otherwise it can lead to side-effects in the future if we forget about this default transition and I _suppose_  it can affect performance (I _suppose_ , transitions are  calculated under the hood even if not active).